src/main/java/net/ae97/pircboty/PircBotY.java

```diff
@@ -30,7 +30,7 @@
 
 public class PircBotY implements Comparable<PircBotY> {
 
-    public static final String VERSION = "2.0.0";
+    public static final String VERSION = "2.1.1";
```

Comment:
Don't need to change actually, we handle this ourselves, but I should change it so it's not a version i guess